docs/pages/helm/guides/gcp.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: Running a highly-available Teleport cluster using GCP, GKE and Helm
```

Comment:
Same here - let's make sure the title is shorter and matches the URL link. Let's move this guide to kubernetes-access/guides/ too